app/components/collections/admin_component.html.erb

```diff
@@ -0,0 +1,15 @@
+<section class="admin-section p-3 mb-5">
+  <div class="row">
+    <div class="col">
+      <label for="adminFunctionsSelect" class="form-label admin-functions">Admin functions</label>
+    </div>
+    <div class="col">
+      <select id="adminFunctionsSelect" class="form-select" onchange="this.value !== 'select' ? document.querySelector('#collectionAdminSection').src = this.value : document.querySelector('#collectionAdminSection').innerHTML = ''">
+        <%= options %>
+      </select>
+    </div>
+  </div>
+  <div class="row pt-3">
+    <turbo-frame id="collectionAdminSection"></turbo-frame>
+  </div>
+</section>
```

Comment:
Mirrors the HTML from the works admin component except for a diff turbo-frame ID.